packages/wrangler/src/__tests__/r2/helpers.test.ts

```diff
@@ -16,63 +13,56 @@ describe("event notifications", () => {
-							actions: [
-								"PutObject",
-								"CompleteMultipartUpload",
-								"CopyObject",
-								"DeleteObject",
-							],
+							actions: ["PutObject", "CopyObject", "DeleteObject"],
 						},
 						{
+							ruleId: "5aa280bb-39d7-48ed-8c21-405fcd078192",
 							actions: ["DeleteObject"],
 						},
 					],
 				},
-				[crypto.randomUUID()]: {
-					queue: "be6b6a37-ae49-4eea-9032-5e8d3ad1d29b",
+				{
+					queueId: "be6b6a37-ae49-4eea-9032-5e8d3ad1d29b",
+					queueName: "my-queue-2",
 					rules: [
 						{
+							ruleId: "c4725929-3799-477a-a8d9-2d300f957e51",
+							createdAt: "2024-09-05T01:02:03.000Z",
 							prefix: "//1",
 							suffix: "2//",
-							actions: ["DeleteObject"],
+							actions: ["LifecycleDeletion"],
 						},
 					],
 				},
-			},
+			],
 		};
 		const tableOutput = await tableFromNotificationGetResponse(
 			config,
-			response[bucketName],
-			vi
-				.fn()
-				.mockImplementation((_: Pick<Config, "account_id">, queue: string) => ({
-					queue_name: queueMap[queue],
-				}))
+			response
 		);
 		logger.table(tableOutput);
 
 		await expect(std.out).toMatchInlineSnapshot(`
-		"┌────────────┬────────┬────────┬─────────────────────────────┐
-		│ queue_name │ prefix │ suffix │ event_type                  │
-		├────────────┼────────┼────────┼─────────────────────────────┤
-		│ my-queue-1 │ /p1    │ s1/    │ object-create,object-delete │
-		├────────────┼────────┼────────┼─────────────────────────────┤
-		│ my-queue-1 │        │        │ object-delete               │
-		├────────────┼────────┼────────┼─────────────────────────────┤
-		│ my-queue-2 │ //1    │ 2//    │ object-delete               │
-		└────────────┴────────┴────────┴─────────────────────────────┘"
+		"┌──────────────────────────────────────┬──────────────────────────┬────────────┬────────┬────────┬───────────────────────────────────┐
+		│ rule_id                              │ created_at               │ queue_name │ prefix │ suffix │ event_type                        │
+		├──────────────────────────────────────┼──────────────────────────┼────────────┼────────┼────────┼───────────────────────────────────┤
+		│ 68746106-12f8-4bba-a57b-adaf37fe11ca │                          │ my-queue-1 │ /p1    │ s1/    │ PutObject,CopyObject,DeleteObject │
+		├──────────────────────────────────────┼──────────────────────────┼────────────┼────────┼────────┼───────────────────────────────────┤
+		│ 5aa280bb-39d7-48ed-8c21-405fcd078192 │                          │ my-queue-1 │        │        │ DeleteObject                      │
+		├──────────────────────────────────────┼──────────────────────────┼────────────┼────────┼────────┼───────────────────────────────────┤
+		│ c4725929-3799-477a-a8d9-2d300f957e51 │ 2024-09-05T01:02:03.000Z │ my-queue-2 │ //1    │ 2//    │ LifecycleDeletion                 │
+		└──────────────────────────────────────┴──────────────────────────┴────────────┴────────┴────────┴───────────────────────────────────┘"
```

Comment:
This table is getting quite wide. Would it make sense to switch to a labelled list? (like `deployments list`)

Easily done with the `formatLabelledValues` util

Reply:
Yes I like this suggestion, I'll check it out.